pxr/imaging/hdSt/shaders/terminals.glslfx

```diff
@@ -10,6 +10,25 @@
 --- This is what an import might look like.
 --- #import $TOOLS/hdSt/shaders/terminals.glslfx
 
+--- --------------------------------------------------------------------------
+-- glsl TessEval.CustomDisplacement
+vec3 DisplacementTerminal(int i0, int i1, int i2, int i3, vec3 Peye, vec3 Neye, vec4 basis, vec2 uv)
+{
+    vec2 t0 = HdGet_st(i0);
+    vec2 t1 = HdGet_st(i1);
+    vec2 t2 = HdGet_st(i2);
+    vec2 t3 = HdGet_st(i3);
+    vec2 t = InterpolatePrimvar(t0, t1, t2, t3, basis, uv);
```

Comment:
The signal driving displacement might not be named `st`, e.g. an asset like `usdImagingGL/testenv/testUsdImagingGLDisplacement/testUsdImagingGLDisplacement.usda` will fail during shader compilation since the displacement input coord is named `st_faceVarying`.

Reply:
I see. I'm not quite sure what a comprehensive solution would look like. Will have to think about that.

Reply:
Yeah, that part of the codeGen texture accessors is delicate. I've been looking into refactoring the displacement terminal a little differently than you have here and I'll tty to share that back with you (possibly next week).

Reply:
That will help, thanks. I appreciate that.